### PR TITLE
Prepare release 1.3.0.

### PR DESCRIPTION
**Pull request by YPCrumble**
Drop support for Python < 3.8 and Django < 3.2.

### CODE REVIEW

**Comment by YPCrumble** [comment]
@peterbe let me know if you need anything else with this?

**Comment by YPCrumble** [comment]
@peterbe I found some energy and think this should work for supporting Django 4.1 and 4.2 as well while maintaining support for 3.2.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
         # PyPy disabled per issue #69:
         # https://github.com/peterbe/django-fancy-cache/issues/69
         # python-version: [3.7, 3.8, 3.9, "3.10", "pypy-3.8"]
-        python-version: [3.7, 3.8, 3.9, "3.10"]
+        python-version: [3.7, 3.8, 3.9, "3.10", 3.11]
```
Comment (peterbe) [inline .github/workflows/ci.yml:17]:
```suggestion
        python-version: [3.7, 3.8, 3.9, "3.10", "3.11"]
```

**Review by peterbe — APPROVED** [review]

**Comment by peterbe** [comment]
@YPCrumble Can you check if https://pypi.org/project/django-fancy-cache/1.3.0/ worked?

**Pull request merged**